src/probnum/filtsmooth/bayesfiltsmooth.py

```diff
@@ -36,53 +36,3 @@ def smoother_step(self, **kwargs):
             + "the Bayesian smoother {}.".format(type(self).__name__)
         )
         raise NotImplementedError(errormsg)
-
-    def predict(self, start, stop, randvar, **kwargs):
```

Comment:
because it is quite cumbersome to force a particle filter into predict and update. I know that the books define a filter like that, but in code, a PF looks different and predict/update is only sensible for Gaussian filters I'd argue.

Reply:
Thanks for clarifying :)